python/cugraph/cugraph/structure/property_graph.py

```diff
@@ -746,8 +832,16 @@ def add_edge_data(
 
     def get_edge_data(self, edge_ids=None, types=None, columns=None):
         """
-        Return a dataframe containing edge properties for only the specified
+        Create a dataframe containing edge properties for only the specified
```

Comment:
Looks like you changed "Return" to "Create", but shouldn't it be "Return"?

Reply:
fixed